apps/emqx/src/emqx_cm.erl

```diff
@@ -96,6 +102,11 @@
 
 -define(T_TAKEOVER, 15000).
 
+%% linting overrides
+-elvis([ {elvis_style, invalid_dynamic_call, #{ignore => [emqx_cm]}}
+       , {elvis_style, god_modules, #{ignore => [emqx_cm]}}
```

Comment:
I'm wondering if we should heed this warning or just ignore it indeed. :thinking: